provider/resources.go

```diff
@@ -1241,13 +1239,11 @@ func Provider() tfbridge.ProviderInfo {
 			"google_compute_region_ssl_policy":                          {Tok: gcpResource(gcpCompute, "RegionSslPolicy")},
 			"google_compute_resource_policy":                            {Tok: gcpResource(gcpCompute, "ResourcePolicy")},
 			"google_compute_route":                                      {Tok: gcpResource(gcpCompute, "Route")},
-			"google_compute_router":                                     {Tok: gcpResource(gcpCompute, "Router")},
+			"google_compute_router":                                     {Fields: lowercaseAutoNameFields()},
```

Comment:
Did you leave out the `Tok` field on purpose? It looks like auto token mapping takes care of this - but then should we be consistent and remove the explicit token mapping from `Network` and the other refactored resources as well?

Reply:
> Did you leave out the Tok field on purpose?

Yes

> It looks like auto token mapping takes care of this - but then should we be consistent and remove the explicit token mapping from `Network` and the other refactored resources as well?

Sure.